src/lib.rs

```diff
@@ -21,6 +26,11 @@ pub use resolver::{resolve_within, BrokenReference};
 pub mod resolved;
 mod resolver;
 
+#[cfg(not(any(feature = "schemars08", feature = "schemars09", feature = "schemars1")))]
+compile_error!(
+    "Any of the features `schemars08`, `schemars09` or `schemars1` must be enabled for this crate."
+);
+
```

Comment:
Features should be additive.
If we wanted to support multiple types (from different crate (versions)), then the right thing to do is abstract over that with a trait.

However, I think the right thing to do here is to force `schemars1` - it's hit major release

Reply:
Fixed.